### ISSUE TITLE
pre-trained model

### ISSUE THREAD

**Issue by tornadomeet** [Conchylicultor/DeepQA#1]
Hello, @Conchylicultor 
could you provide a pre-trained model for fun?
thanks~

**Comment by Conchylicultor** [comment]
Right know, the longer model I've trained still mainly gives evasive answer. If you want, I could still upload it on Drive but don't expect spectacular results. I'm just using my personal laptop for training so I'm quite limited by my hardware.

**Comment by tornadomeet** [comment]
thanks, you can upload it on Drive :) 
i have gpus, i'll train the model when my  gpu is idle.

**Comment by Conchylicultor** [comment]
Here [you go](https://drive.google.com/file/d/0Bw-phsNSkq23amlSZXVqcm5oVFU/view?usp=sharing). Just decompress it on the save folder and run it with:

```
python main.py --modelTag pretrained --test interactive
```
You can see the parameters for which the model has been trained on the params.ini file. Here, I'm mainly limited by the word embedding size and probably also the hidden dimension. To get you an idea, the original seq2seq paper speak about a 5 layers LSTM with embedding and hidden size of 1000. But after, the deeper the model is, the more difficult it should be to optimize it.

**Comment by tornadomeet** [comment]
@Conchylicultor i just download your model and tried it, it a cool work!  Yes, i think i will train a model for fun~
thanks.

**Comment by tornadomeet** [comment]
@Conchylicultor  i'm start training a more complex model now, the trainning speed in k80 is ok, cmd is ```python -u main.py --embeddingSize 128 --hiddenSize 512 --numLayers 5 --batchSize 128 --maxLength 20```

**Comment by Conchylicultor** [comment]
Nice. Thanks for the info :)

**Comment by tornadomeet** [comment]
@Conchylicultor  i have finished the model(about 300 epoch), the loss is about < 0.9, here is i talk with it.
![chatbot](https://cloud.githubusercontent.com/assets/5860892/18421787/bacbf9aa-78c4-11e6-88c8-abcdcea47ead.jpg)

**Comment by tornadomeet** [comment]
i upload the trained  model to dropbox, https://www.dropbox.com/sh/yii4h2hmz4ri0zt/AABwqknB1fbgXPi7mQnATuVDa?dl=0 when it finished it uploading, we can use it for fun.
ps: i found when we input the question, we'd better input the punctuation together.

**Comment by Conchylicultor** [comment]
Thank you very much, it's nice to see the program generate longer sentences. Yes, I notice the punctuation too. It's certainly linked to the format of the training samples.

**Comment by qwaider** [comment]
Do you still have this model, the google drive model is doing really bad , generating long sentences without meaning or sense

**Comment by niazangels** [comment]
@qwaider Did you try renaming the folder in `save` to `model` instead of `model-pretrainedv2` ?

**Comment by niazangels** [comment]
@tornadomeet Would love to test your trained model :)  
The Dropbox link seems to be down- could you please share the model again if you still have it?

**Comment by celikmustafa89** [comment]
I think all pre-trained models have broken links. Is there any, can you share it?

**Comment by julien-c** [comment]
i can suggest using huggingface.co to upload models, if it can helps! 

(I was using DeepQA in 2016 and loved it :)
